torch/_torch_docs.py

```diff
@@ -6010,6 +6009,60 @@ def merge_dicts(*dicts):
             [ 0.1807]])
 """.format(**multi_dim_common))
 
+add_docstr(torch.nanmean, r"""
+nanmean(input) -> Tensor
```

Comment:
Can we document this signature in one function? The double signature thing is lame and an artifact of imperfect op implementations